src/Umbraco.Web.Common/Authorization/AuthorizationPolicies.cs

```diff
@@ -53,6 +53,7 @@ public static class AuthorizationPolicies
     public const string TreeAccessMediaOrMediaTypes = nameof(TreeAccessMediaOrMediaTypes);
     public const string TreeAccessDictionaryOrTemplates = nameof(TreeAccessDictionaryOrTemplates);
     public const string TreeAccessDocumentOrMediaOrContentTypes = nameof(TreeAccessDocumentOrMediaOrContentTypes);
+    public const string TreeAccessDocumentOrMediaOrContentTypesOrMembers = nameof(TreeAccessDocumentOrMediaOrContentTypesOrMembers);
```

Comment:
Should probably be `Member` in singular as `Document` and `Media`:

`TreeAccessDocumentOrMediaOrContentTypesOrMember`

But it doesn't quite seem consistent in naming, e.g. `Templates` and `ContentTypes`

And we have `TreeAccessStylesheetsOrDocumentOrMediaOrMember` and `TreeAccessMembersOrMemberTypes`.

But for this on perhaps `TreeAccessDocumentOrMediaOrMemberOrContentTypes` is better, which is more consistent with `TreeAccessStylesheetsOrDocumentOrMediaOrMember` mainly the `DocumentOrMediaOrMember` part 😅

Reply:
Good point @bjarnef 💪 
I'll create a separate task for aligning the policy names to not grow the scope of this PR 🙂 

I can see that we are using more plural forms so until the policy names refactoring, we will keep `TreeAccessDocumentsOrMediaOrMembersOrContentTypes`